python/podio/version.py

```diff
@@ -1,6 +1,9 @@
 #!/usr/bin/env python3
 """Module that facilitates working with the podio::version::Version"""
 
+from functools import total_ordering
+from packaging import version
```

Comment:
Should this also appear in the `requirements.txt` (and readme?)?

Reply:
Indeed it should. I though it was part of the standard library.